library/spdm_requester_lib/libspdm_req_get_certificate.c

```diff
@@ -158,11 +162,20 @@ return_status try_spdm_get_certificate(IN void *context, IN uint8_t slot_id,
             status = RETURN_DEVICE_ERROR;
             goto done;
         }
+        if (spdm_request.offset == 0) {
+            total_responder_cert_chain_buffer_length = spdm_response.portion_length + spdm_response.remainder_length;
+        }
+        else if ((spdm_request.offset + spdm_response.portion_length + spdm_response.remainder_length != total_responder_cert_chain_buffer_length)
+                 || (spdm_response.portion_length <= 0)) {
```

Comment:
Portion_length is uint16_t, it cannot < 0.
Please change to (spdm_response.portion_length == 0).

Reply:
Yes. Even if the responder sends negative values, the values requester read is positive values. And I have changed the code location to prevent stop for first GET_CERTIFICATE request.